src/database/DatabaseConnector.java

```diff
@@ -0,0 +1,62 @@
+package database;
+
+import java.sql.*;
+
+public class DatabaseConnector {
+	
+	private final String DEFAULT_DATABASE = "question_library";
+	
+	private DatabaseErrorHandler errorhandler;
+	
+	private Connection con;
+
+	public DatabaseConnector() {
+		errorhandler = new DatabaseErrorHandler();
+	}
+
+	/**
+	 * Establishes a Connection to the Database
+	 * @param databasename The Name of the database to connect to
+	 * @return true, if the connection was successfully established
+	 * or false, if an Exception occurred
+	 */
+	public boolean establishConnection(String databasename) {
```

Comment:
I dislike the idea of having the object in an unusable state (an **unconnected** state). It should connect on instantiation.